doc/ref_model/chapters/chapter03.md

```diff
@@ -287,5 +287,21 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
 <a name="3.8.2"></a>
 ### 3.8.2 Levels of accelerator consumption
 
-<a name="3.8.3"></a>
-### 3.8.3 Decoupling applications from infrastructure with Hardware Acceleration
+<a name="3.8.9"></a>
+### 3.8.9 Decoupling Applications from Infrastructure and Platform with Hardware Acceleration
+
+[Decoupling](https://github.com/cntt-n/CNTT/blob/master/doc/common/glossary.md#cloud-platform-abstraction-related-terminology) application from hardware accelerator normally goes through drivers that if available are preferred to be towards some interface standardized across vendors and their products, or if not available then through drivers specific to hardware device”. Decoupling infrastructure software from hardware accelerator is also preferred through standardized interface, or if not available is comparatively less of an issue for one or few software infrastructure functions than similar for many applications.
+
+Taking advantage of RM and RA environments with common capabilities, applications can be developed and deployed more rapidly, providing more service agility and easier operations. The extent to which this can be achieved will depend on levels of decoupling between application and infrastructure or platform underneath the application:
+
+#### 1. Infrastructure:
+- a) Application functionality or application control requires infrastructure components beyond RM profiles or infrastructure configuration changes beyond RA-exposed APIs. Generally, such an application is tightly coupled with the infrastructure which results in an [Appliance deployment model](https://github.com/cntt-n/CNTT/blob/master/doc/common/glossary.md#cloud-platform-abstraction-related-terminology).
+- b) Application control using RA APIs finds node (already configured in support of the profiles) with required infrastructure component(s), and in that node using RA APIs configures infrastructure components that make application work. Example is application that to achieve latency requirements needs certain hardware acceleration available in RM profile and is exposed through RA APIs.
```

Comment:
Do we have "RA API"-s? I think we do not, but we have API-s required by RA2.

Reply:
> 
> 
> Do we have "RA API"-s? I think we do not, but we have API-s required by RA2.

Changed. Thanks